easycorp/easy-deploy-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,6 @@
+{
+    "bundles": {
+        "EasyCorp\\Bundle\\EasyDeployBundle\\EasydeployBundle": ["dev"]
```

Comment:
Yes, that's a typo ... but the error (related to the bundle? to Flex?) persists:

```
composer create-project symfony/skeleton:3.3.x-dev demo
cd demo/
composer require --dev easycorp/easy-deploy-bundle

!!  PHP Fatal error:  Uncaught Symfony\Component\Debug\Exception\ClassNotFoundException: Attempted to load class "EasyCorpEasyDeployBundle" from namespace "EasyCorp\Bundle\EasyDeployBundle".
!!  Did you forget a "use" statement for another namespace? in <project>/src/Kernel.php:35
!!  Stack trace:
!!  #0 <project>/vendor/symfony/http-kernel/Kernel.php(450): App\Kernel->registerBundles()
!!  #1 <project>/vendor/symfony/http-kernel/Kernel.php(116): Symfony\Component\HttpKernel\Kernel->initializeBundles()
!!  #2 <project>/vendor/symfony/framework-bundle/Console/Application.php(137): Symfony\Component\HttpKernel\Kernel->boot()
!!  #3 <project>/vendor/symfony/framework-bundle/Console/Application.php(124): Symfony\Bundle\FrameworkBundle\Console\Application->registerCommands()
!!  #4 <project>/vendor/symfony/console/Application.php(90): Symfony\Bundle\FrameworkBundle\Console\Application->add(Object(Symfony\Component\Console\Command\HelpCommand))
!!  #5 <project>/vendor/symfony/framework-bundle in <project>/src/Kernel.php on line 35
!!
!!
!!  Fatal error: Uncaught Symfony\Component\Debug\Exception\ClassNotFoundException: Attempted to load class "EasyCorpEasyDeployBundle" from namespace "EasyCorp\Bundle\EasyDeployBundle".
!!  Did you forget a "use" statement for another namespace? in <project>/src/Kernel.php:35
!!  Stack trace:
!!  #0 <project>/vendor/symfony/http-kernel/Kernel.php(450): App\Kernel->registerBundles()
!!  #1 <project>/vendor/symfony/http-kernel/Kernel.php(116): Symfony\Component\HttpKernel\Kernel->initializeBundles()
!!  #2 <project>/vendor/symfony/framework-bundle/Console/Application.php(137): Symfony\Component\HttpKernel\Kernel->boot()
!!  #3 <project>/vendor/symfony/framework-bundle/Console/Application.php(124): Symfony\Bundle\FrameworkBundle\Console\Application->registerCommands()
!!  #4 <project>/vendor/symfony/console/Application.php(90): Symfony\Bundle\FrameworkBundle\Console\Application->add(Object(Symfony\Component\Console\Command\HelpCommand))
!!  #5 <project>/vendor/symfony/framework-bundle in <project>/src/Kernel.php on line 35
!!
```